### DISCUSSION TITLE
Exporting table containing SVGs

### DISCUSSION THREAD

**Discussion by thatsnotmycat** [TIBCOSoftware/spotfire-mods#121] [Q&A]
Hi,

I wrote a spotfire mod, that creates a data table containing histograms in one column.
Sometimes the user needs to export the data (preferably to excel), so that he can add some comments.

Is there anyone who has experience, how to achieve this?
Keeping the Histograms in Excel would be great, but if it's not possible at all, exporting only the table content without the svgs would be ok.

I already tried using the xlsx package, but I think the mod might not have access to the filesystem, so the data isn't going anywhere.

The spotfire built-in export function wouldn't allow me to export the data from visualization (only from the data table)

Thanks a lot in advance

**Comment by CommonCrisis** [comment]
Not possible - mods are really limited
